README.md

```diff
@@ -1,4 +1,7 @@
-#Concat
+#concat
+
+
+[![Build Status](https://travis-ci.org/puppetlabs/puppetlabs-concat.png?branch=master)](https://travis-ci.org/puppetlabs/puppetlabs-concat)
```

Comment:
Will do. Do I read correctly that I should do same in all future README updates?